decidim-meetings/app/models/decidim/meetings/meeting.rb

```diff
@@ -90,7 +90,7 @@ def can_participate?(user)
       end
 
       def can_view_meeting?(current_user)
-        return true unless self.try(:is_private?)
+        return true unless try(:is_private?)
```

Comment:
Why use `try` here? I don't think it's needed, as you're calling the method directly in the next line 😕